spinta/types/model.py

```diff
@@ -210,6 +250,9 @@ def load(
     prop.lang = load_lang_data(context, prop.lang)
     prop.comments = load_comments(prop, prop.comments)
 
+    if data['type'] == 'denorm':
+        _check_denorm_prop(prop.name, prop, prop.model, manifest)
```

Comment:
This should be done in `check` command, because, during `load` not all models might be loaded yet.

Reply:
I saw in testing notes that generated datatypes in database should be fixed, so that they would be the same as in ref model. For that I needed to add rel_prop to Denorm datatype and assign it during link command. And since it was doing pretty much the same thing as _check_denorm_prop, I also moved this check to link.